src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx

```diff
@@ -37,49 +41,88 @@ function getDropdownValueFromLocation(pathname) {
       subPageTitle: 'Requested',
       subHeading: 'Requested appointments',
     };
-  } else if (pathname.endsWith(DROPDOWN_VALUES.past)) {
+  }
+  if (pathname.endsWith(DROPDOWN_VALUES.past)) {
     return {
       dropdownValue: DROPDOWN_VALUES.past,
       subPageTitle: 'Past appointments',
       subHeading: 'Past appointments',
     };
-  } else if (pathname.endsWith(DROPDOWN_VALUES.canceled)) {
+  }
+  if (pathname.endsWith(DROPDOWN_VALUES.canceled)) {
     return {
       dropdownValue: DROPDOWN_VALUES.canceled,
       subPageTitle: 'Canceled appointments',
       subHeading: 'Canceled appointments',
     };
-  } else {
-    return {
-      dropdownValue: DROPDOWN_VALUES.upcoming,
-      subPageTitle: 'Your appointments',
-      subHeading: 'Your appointments',
-    };
   }
+  return {
+    dropdownValue: DROPDOWN_VALUES.upcoming,
+    subPageTitle: 'Your appointments',
+    subHeading: 'Your appointments',
+  };
+}
+
+function handleDropdownChange(history, setHasTypeChanged) {
+  return e => {
+    const { value } = e.target;
+    if (value === DROPDOWN_VALUES.upcoming) {
+      history.push('/');
+    } else if (value === DROPDOWN_VALUES.requested) {
+      history.push('/requested');
+    } else if (value === DROPDOWN_VALUES.past) {
+      history.push('/past');
+    } else if (value === DROPDOWN_VALUES.canceled) {
+      history.push('/canceled');
+    }
+    setHasTypeChanged(true);
+  };
+}
+
+function renderWarningNotification() {
+  return (props, childContent) => {
+    const { status } = props;
+    return (
+      <WarningNotification status={status}>{childContent}</WarningNotification>
+    );
+  };
 }
 
 export default function AppointmentsPageV2() {
   const location = useLocation();
   const [hasTypeChanged, setHasTypeChanged] = useState(false);
-  const showScheduleButton = useSelector(state => selectFeatureRequests(state));
+  const featureStatusImprovement = useSelector(state =>
+    selectFeatureStatusImprovement(state),
+  );
+  const pendingAppointments = useSelector(state =>
+    selectPendingAppointments(state),
+  );
   const {
     dropdownValue,
     subPageTitle,
     subHeading,
   } = getDropdownValueFromLocation(location.pathname);
 
+  const [count, setCount] = useState(0);
   useEffect(
     () => {
-      document.title = `${subPageTitle} | ${pageTitle} | Veterans Affairs`;
+      if (featureStatusImprovement) {
+        let prefix = subPageTitle;
+        if (subPageTitle.startsWith('Request')) prefix = 'Pending appointments';
+        document.title = `${prefix} | VA online scheduling | Veterans Affairs`;
+        pageTitle = 'Your appointments';
```

Comment:
Helps with usability to bring focus to the h1 heading rather than document.body
```suggestion
        pageTitle = 'Your appointments';
        scrollAndFocus('h1');
```

Reply:
Good catch!